app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -72,7 +69,6 @@ def update
     api :POST, '/organizations', N_('Create organization')
     param :name, String, :desc => N_("name"), :required => true
     param :label, String, :desc => N_("unique label")
-    param :description, String, :desc => N_("description")
```

Comment:
@thomasmckay I can't test this because I get the error I mentioned: `Dont know how to merge Must be String with Must be String (NotImplementedError)` unless I remove 
```ruby
     param_group :resource, ::Api::V2::TaxonomiesController
```
which is were the new description param comes from.
Maybe @iNecas or @ehelms will have an idea?